test/integration/libp2p_test.exs

```diff
@@ -0,0 +1,172 @@
+defmodule Libp2pTest do
+  use ExUnit.Case
+  doctest Libp2p
+
+  # Mainnet
+  @bootnodes [
```

Comment:
maybe we want to move these to config? https://hexdocs.pm/elixir/main/Config.html

We can address this later though.